src/test/java/owen/DateTimeTest.java

```diff
@@ -0,0 +1,27 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import owen.DateTime;
+import owen.OwenException;
+
+public class DateTimeTest {
+    @Test
+    public void parse_correctFormat_success() throws OwenException {
+        DateTime dateTime = DateTime.parse("1/1/2021 1400");
+        assertEquals("January 1 2021 2:00 PM", dateTime.toString());
+    }
+
+    @Test
+    public void parse_incorrectFormat_exception() {
+        try {
+            DateTime.parse("January 1 2021 2:00 PM");
+            fail();
+        } catch (OwenException exception) {
+            assertEquals(
+                    "Ooooo noo...\nDate and time must be in DD/MM/YYYY HHMM format...",
+                    exception.getMessage());
+        }
+    }
+}
```

Comment:
Format looks good to me, but perhaps if you have time, you could write a short description of the test cases?